README.md

```diff
@@ -19,58 +19,60 @@ To install the software below we recommend the following package managers:
   1. Once Ruby is installed, run `gem install bundler` to install.
 1. PostgreSQL v9.2+ (database)
 1. ImageMagick
+1. Set up a MESS OAuth client, if you don't have one, on Google Developers.
 
 ## Development Setup Guide
 Follow these steps to setup a development environment for MESS.
 
 1. **Install all above dependencies**
 
 1. **Retrieve project files using Git**
-
   ```
   git clone https://github.com/touchstone-cohousing/mess.git
   cd mess
   ```
 
   If developing, it's best to work off the development branch:
-
   ```
   git checkout develop
   ```
 
+  The remaining steps should all be done from the project directory. 
+
 1. **Install gems**
-  - Install the required gems by running `bundle install` in the project directory.
+  - Run `bundle install` to install the required gems.
 
 1. **Set local config**
-  - `cp config/initializers/local_config.rb.example config/initializers/local_config.rb`
-  - Edit `config/initializers/local_config.rb` to set config specific to your environment.
-
-1. **Create and setup development and test databases**
-  - See `createdb` command.
-  - Should be named `mess_development` and `mess_test`.
-  - Ensure adquate privileges for table creation, etc.
-  - Setup tables: `bundle exec rake db:schema:load`.
-
-1. **Seed the db**
-  - `rake db:seed`
-  - This adds communities, households, etc.
-
-1. **Create a user so you can log in**
-  - Open the rails console (`rails c`)
-  - Type e.g. `User.create!(email: "you@example.com", google_email: "you@gmail.com", first_name: "Jane", last_name: "Doe", mobile_phone: "17345551212", household: Household.first, admin: true)`
-  - Be sure to replace `you@gmail.com` with a Google account you control.
+  - Copy `config/initializers/local_config.rb.example` to `config/initializers/local_config.rb`.
+  - Edit `config/initializers/local_config.rb` as follows:
+      Get `<client ID>` and `<client secret>` from your Google Developers MESS OAuth client.
+      Get a `<secret key>` by running `rake secret`.
+      Replace `<hostname>` with your web and smtp servers. 
+
+1. **Create development and test databases and schemas**
+  - Copy `config.database.yml.example` to `config.database.yml`.
+  - Run `rake db:create` to create `mess_development` and `mess_test` databases.
+  - Run `rake db:schema:load` to create the schema in both databases.
+
+1. **Seed the development db**
+  - Copy `db/seeds.rb.example` to `db/seeds.rb`.
+  - Edit `db/seeds.rb`, replacing `<your gmail name>` with your GMail name.
+  - Run `rake db:seed` to add one community, one household, and one user (with admin privileges and your GMail address) to the mess_development database.
```

Comment:
I get it about seed data should be applicable to production as well.  

However, just removing this section leaves the developer with no way to log into MESS.  And the prior instructions (steps 6 & 7 at https://github.com/touchstone-cohousing/mess) don't work, so just restoring them is not an option.  Specifically, running rake db:seed (step 6) does nothing if there's no seeds.rb script; and you can't create a user (step 7) without first creating a community and a household for the user to be a part of.

So I think I will work on creating a fake_data rake task as you suggest.  (Also, this looks like a better apprentice-learning project than fixing a bug!)